### DISCUSSION TITLE
Add more infos to the games

### DISCUSSION THREAD

**Discussion by cosmo0** [benbarden/switchscores#47] [Ideas]
*Copied from https://github.com/benbarden/switchscores/issues/38*

Here are a few ideas (some taken from other sites) of data that would enrich the user experience:

From Nintendo API:

**Age rating**, & descriptions (violence...) if possible:  it's useful for people who look for games they can play with their kids. It's not a end-all-be-all to whether a game is appropriate for kids: hardcore racing sims are rated PEGI 3 because they don't feature violence or anything; and each kid is sensitive to different things. But it can eliminate violent or scary games when you're looking for something to play with your 5-yo.

**Controllers**: not sure it's very useful

**Features** (amiibo, handheld...): not sure it's very useful either

**Languages**: In many countries, a lot of people have enough english knowledge to browse a website, but not enough to play a full game. I see a lot of people refusing to play very good games just because they don't have french language option.

**Description**: if only the very small excerpt is in the API, I'm not sure it's worth the trouble. Maybe leave it in favor of when/if a page scrapping mechanism is developed.

**Demo**: could get people interested in games they would otherwise skip because they're not sure

**Youtube trailer**: SensCritique (a french "Metacritic 100% by users") has a wiki-like feature to edit pages, and users can add trailers links to Youtube page: [example](https://user-images.githubusercontent.com/3973850/105617029-f08e2300-5ddb-11eb-963c-d0d78ea5713b.png). It would require users to fill these out (if they exist, which they often do), but it would allow you to easily insert videos in the page, without hosting anything, without upload, etc, just embed the Youtube player: more info for the user, not too much work for you.

You've also talked about multiplayer modes, what kind of info is there in this field?

---

Links to other sites:

- Steam, with a review summary and link (see #16 ). Could be manually added by users.
- Metacritic: some games are released on Switch a long time after other platforms, and some very good games have zero reviews on SwitchScore (ex: Firewatch). Could be added manually as an alternative, only when the game is not ranked?
- Link to Howlongtobeat, maybe added manually.
- Link to price history (DekuDeals or Psprices), maybe added manually.

I totally understand that you don't want to clutter the site, but not everyone know about Howlongtobeat or price history sites, so I still think at least linking to them would be beneficial to the users.

**Comment by cosmo0** [comment]
@benbarden here's a quick mockup for displaying more infos in the lists: https://codepen.io/cosmo0/full/BaLXzZv

I don't think it's perfect (it's too cluttered and probably ugly when integrated inside the actual pages) but hopefully it might give you an idea?

**Comment by benbarden** [comment]
I'm not sure ... I thought it might work, but if you look at the size of most images on the homepage (except for the bigger images at the top), I don't think there's a lot of space for more info. But most of it could go on the game pages anyway.

I did wonder if the release date needs to be shown on every page though (as it is now). That might free up some space.

**Reply by cosmo0** [reply]
I like adding icons but I tend to create cluttered interfaces lol

Here's another, more compact view: https://codepen.io/cosmo0/full/NWRQoBy

Yes maybe release date is not that important to show on the games lists. The category would be much more useful.

**Reply by benbarden** [reply]
This is better for sure. I think the age rating looks out of place as it doesn't fit with the theme. That might be one to move to the game page and leave off the lists.
